website/description_issues.qmd

```diff
@@ -135,3 +135,36 @@ The Maintainer and Author fields are automatically generated, so you don't need
 If your package uses the older approach of specifying Maintainer and Author separately in the DESCRIPTION file, it won’t be rejected for that reason alone. However, this method is discouraged, and any manual modifications that differ from the auto-generated fields will result in automatic rejection by CRAN.
 
 :::
+
+
+------------------------------------------------------------------------
+
+# References
+
+## Problem
+
+The reference link in the Description field is incorrectly formatted and isn't clickable.
```

Comment:
```suggestion
The reference link in the Description field is incorrectly formatted or isn't clickable.
```

Changed it to `or` since sometimes the linking is correct, but the authors/year are displayed inconsistently.